lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -66,7 +66,7 @@ def __init__(
         self.password = password
         self.database = database
 
-    def _connect_to_database(self, database: str = None) -> psycopg2.extensions.connection:
+    def connect_to_database(self, database: str = None) -> psycopg2.extensions.connection:
```

Comment:
I guess you need to bump this lib version, right?

Reply:
We are going to bump that lib version in the PostgreSQL k8s charm. There is also some changes that were made in the lib through the PostgreSQL VM charm that we will need to consolidate.